### ISSUE TITLE
Can't fill in talents and equipment

### ISSUE THREAD

**Issue by EctoNoodle** [pwatson100/alienrpg#29]
Hey, I just bought FoundryVTT in order to play Alien RPG (I have both the physical book and the PDF version)
When trying to make characters I can not fill in the talents and I can also not fill in the equipment.

**Comment by pwatson100** [comment]
Hi, So with items equipment ect. You need to create them as Items then drag and drop them on to the character sheet.  They will then appear in the correct place (Talents, Agendas or Equipment) and can be edited (for that player) by right clicking on the edit icon on the right of the item.  
If you have not already spotted it un Compendiums under Journals you should see "Mother Instructions" where I have tried to give an outline of how things work.   If you are on Discord you are always welcome to ping me on there if you need any help.  On the Foundry Dicsord there is a section where we discuss all Free League games that are available on Foundry VTT which you might find useful.

**Comment by EctoNoodle** [comment]
Yeah, sorry for bothering you with this, like I said I just purchased it today just to play Alien RPG. After some deeper digging I found the mother instructions and managed to figure it out. I'm still new to this kind of thing so it wasn't as obvious to me as it might have been to someone else :)

Also this Alien RPG module thingy you made is AMAZING!
Only thing I would have liked was if the character sheet layout would have been a little more in the same style of the original, but that's just me :)

**Comment by pwatson100** [comment]
It's really not a problem I am glad to help.  We were all new once.  
Thank you for the feedback, it's really appreciated and I am glad you are enjoying the system.  

I did think about making the character sheet similar to the one in the books but there were two issues:
1. I can't reproduce anything from the books in the system.  Free League were kind enough to let me use the Panic Table as I explained the system would be pretty useless without it.  
2. It's really hard to get all the necessary information using the official sheet format.  There is a lot of stuff to record that is not really on their sheet (I end up writing on the back of them). 

So I took the easier option :)
I tried to capture the style of the monitors from the films.
